Move mute button to master volume circle

peppy opened the pull request:
A proposal to solve https://github.com/ppy/osu/discussions/29344. And to move the mute button to somewhere actually visible/sensible.

Doesn't look great but the whole volume metres are going to get a redesign so maybe a good first step? @arflyte does the new design have a mute button implemented somewhere?

peppy committed:
Move mute button to master volume circle

smoogipoo commented:
How about making the icon a liiiiiiiittle bit smaller?

Plus, it looks like it's probably overlapping the border here and adding weird white corners (I don't even want to know 🙈):
<img width="88" alt="image" src="https://github.com/user-attachments/assets/f59b926e-c8d6-4fc8-8f66-545cb2c54183">

peppy commented:
Just the icon, not the circle right?

peppy committed:
Make icon smaller

smoogipoo commented:
Yeah just the icon, it felt too full anyway.

smoogipoo committed:
Hack around the border looking ugly

This is an o!f issue because borders are applied into the individual
sprites of the container via masking, rather than being isolated to the
container itself. In this case, it'll be applied to the "flash" sprite,
which is using additive blending, causing further issues.
